be/src/column/const_column.h

```diff
@@ -113,10 +113,10 @@ class ConstColumn final : public ColumnFactory<Column, ConstColumn> {
     bool append_nulls(size_t count) override {
         if (_data->is_nullable()) {
             bool ok = true;
-            if (_size == 0) {
+            if (count > 0 && _size == 0) {
```

Comment:
1. I think the caller should make sure the count large than zero.  we could and dcheck even check for the count > 0 
2. count > 0 & _size == 0 maybe faster than count > 0 && _size == 0